src/main/java/com/marklogic/rest/util/MgmtResponseErrorHandler.java

```diff
@@ -49,6 +50,14 @@ public void handleError(ClientHttpResponse response) throws IOException {
 				logger.error(message);
 			}
 			throw ex;
+		} catch (InvalidMediaTypeException ex) {
+			// In at least one scenario - when deleting a REST API server whose modules database has been set to be
```

Comment:
This causes the error to be exposed to the user, which is currently not happening due to the invalid MIME type from MarkLogic, which causes an InvalidMediaTypeException, thus obscuring the actual error.